scripts/apps/search/components/Item.tsx

```diff
@@ -198,6 +199,10 @@ export class Item extends React.Component<IProps, IState> {
     }
 
     dbClick(event) {
+        if (!isMediaEditable(this.props.item)) {
```

Comment:
What happens if this function is not here?

Reply:
Then it will be an attempt to fetch and edit the item. I think it makes sense to put it a little bit lower, to prevent `open` item on double click.

Reply:
It'd be good to move it closer to the actual logic where fetching/editing actually happens.

Reply:
Got it. I've put it before actually to `ItemList.dbClick`, but after, I somehow decided that  `Item.dbClick` is a better candidate for it. Can you explain to me why it's better to move it to `ItemList.dbClick`? In this case, It would be logical to move this condition too:
```
if (!this.props.item.gone) {
        this.props.onDbClick(this.props.item);
 }
```

Reply:
> Can you explain to me why it's better to move it to `ItemList.dbClick` ?

Because if someone called `ItemList.dbClick` and forgot to do the validation, we'd have a bug. It's not too likely at the moment that someone will call it from another place, but when time passes, code gets renamed and restructured and it becomes not that unlikely anymore.

You should probably move the check not to the top of `ItemList.dbClick`, but where the fetching actually happens which is probably [here](https://github.com/superdesk/superdesk-client-core/blob/develop/scripts/apps/search/components/ItemList.tsx#L242)

I agree `this.props.item.gone` check should probably be moved too.